src/App.tsx

```diff
@@ -48,34 +67,71 @@ export function getReorderedGoods(
 // };
 
 export const App: React.FC = () => {
+  const [isReversed, setReversed] = React.useState(false);
+  const [sortType, setSortType] = React.useState(SortType.NONE);
+
+  const alphabetLight = classNames({
+    'is-light': sortType !== SortType.ALPHABET,
+  });
+  const lengthLight = classNames({
+    'is-light': sortType !== SortType.LENGTH,
+  });
+  const reverseLight = classNames({
+    'is-light': !isReversed,
+  });
+
+  const vResetButton = isReversed || sortType !== SortType.NONE;
+
   return (
     <div className="section content">
       <div className="buttons">
-        <button type="button" className="button is-info is-light">
+        <button
+          type="button"
+          className={`button is-info ${alphabetLight}`}
```

Comment:
You can use it like that 
```suggestion
className={classNames('button is-info', {'is-light': sortType !== SortType.ALPHABET})}
```